README.md

```diff
@@ -109,4 +109,6 @@ videojs.Html5DashJS.beforeInitialize = function(player, mediaPlayer) {
     });
   }
 };
+
+videojs.Html5DashJS.registerInitializationHook(myCustomCallback);
```

Comment:
Perhaps a user might want to change dashjs player options on a source by source basis?